interfaces/builtin/account_control_test.go

```diff
@@ -34,23 +36,39 @@ type AccountControlSuite struct {
 	plug  *interfaces.Plug
 }
 
-var _ = Suite(&AccountControlSuite{
-	iface: builtin.NewAccountControlInterface(),
-	slot: &interfaces.Slot{
+var _ = Suite(&AccountControlSuite{})
+
+const mockPlugSnapInfo = `name: other
```

Comment:
Ok, renamed to reflect interface name (I applied this convention already in all other tests, just missed it in this one as it was the first I worked on).